curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -1594,7 +1667,6 @@ void DataCache::Write(uint64_t chunkPos, uint64_t len, const char *data,
                 */
                 if (chunkPos + len <
                     (*iter)->GetChunkPos() + (*iter)->GetLen()) {
-                    chunkCacheManager_->rwLockWrite_.WRLock();
```

Comment:
you just remove this lock, and `chunkCacheManager_->UpdateWriteCacheMap` won't hold lock when modify `dataWCacheMap_`, is this correct?